src/CashuWallet.ts

```diff
@@ -128,15 +125,12 @@ class CashuWallet {
 			}
 		}
 		return {
-			proofs,
+			token: { token: [{ proofs, mint: this.mint.mintUrl }] },
```

Comment:
what is the benefit of doing it this way?

Reply:
https://github.com/cashubtc/cashu-ts/issues/43 ?

Reply:
yeah but we're still only returning tokens from one mint here, right? It would only make sense if we return all tokens.

but also: https://github.com/cashubtc/cashu-ts/pull/46#issuecomment-1527733894

maybe we should do it differently.

Reply:
OK